packages/compartment-mapper/test/scaffold.js

```diff
@@ -58,6 +58,7 @@ export function scaffold(
     shouldFailBeforeArchiveOperations = false,
     knownFailure = false,
     tags = undefined,
+    candidateSuffixes = undefined,
```

Comment:
Perhaps for the public API, using the name `extensions` or just `suffixes` would be more discoverable. Or `searchSuffixes`?

Reply:
i think its fine to use `searchSuffixes` throughout